assets/queries/cloudFormation/viewer_protocol_only_https_compliant/query.rego

```diff
@@ -0,0 +1,28 @@
+package Cx
+
+CxPolicy [ result ] {
+  resources := input.document[i].Resources[name]
+  resources.Properties.DistributionConfig.DefaultCacheBehavior.ViewerProtocolPolicy != "https-only"
+  
+
+	result := {
+                "documentId": 		    input.document[i].id,
+                "searchKey": 	        sprintf("Resources.%s.Properties.DistributionConfig.DefaultCacheBehavior.ViewerProtocolPolicy", [name]),
+                "issueType":		      "IncorrectValue",
+                "keyExpectedValue":   sprintf("Resources.%s.Properties.DistributionConfig.DefaultCacheBehavior.ViewerProtocolPolicy is 'https-only'", [name]),
+                "keyActualValue": 	  sprintf("Resources.%s.Properties.DistributionConfig.DefaultCacheBehavior.ViewerProtocolPolicy 'isn't https-only'", [name])
+              }
+}
+
+CxPolicy [ result ] {
+  resources := input.document[i].Resources[name]
+  resources.Properties.DistributionConfig.CacheBehaviors.ViewerProtocolPolicy != "https-only"
```

Comment:
Cache Behaviors is a list. Replace with:
`resources.Properties.DistributionConfig.CacheBehaviors[_].ViewerProtocolPolicy != "https-only"`